gdcdictionary/schemas/treatment.yaml

```diff
@@ -634,9 +666,12 @@ properties:
       - Radiation, Combination
       - Radiation, Cyberknife
       - Radiation, External Beam
+      - Radiation, Hypofractionated
       - Radiation, Implants
       - Radiation, Intensity-Modulated Radiotherapy
       - Radiation, Internal
+      - Radiation, Mixed Photon Beam
+      - Radiation, Photon Beam
```

Comment:
not sure if it matters, but _Radiation, Photon Beam_ shows up twice (lines 674 and 675)

Reply:
@wteouchicago Proton and Photon are two different types of radiation, so we need both of them. Thanks for checking on this!

Reply:
Ah thanks! I clearly need my eyes checked!